geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -7130,34 +7130,34 @@ public void releaseDestroyLock() {
   @Override
   public void cleanupFailedInitialization() {
     // mark as destroyed fixes 49555.
-    this.isDestroyed = true;
+    isDestroyed = true;
 
     // after isDestroyed is set to true call removeResourceListener to fix bug 49555
-    this.cache.getInternalResourceManager(false).removeResourceListener(this);
+    cache.getInternalResourceManager(false).removeResourceListener(this);
 
     // fixes bug 41333
```

Comment:
I assume the code around here was touched with search and replace but if you were in this area of code manually I would like to see comments like this stripped out. If there is some context from the bug that needs to be expressed here in a meaningful way then I would replace the comment. Generally though these comments just need pruning.

Reply:
I'll try to strip out non-useful comments soon as well. This was IntelliJ automating the removal of unnecessary uses of this.

Reply:
With all of the prechecks green and the fact this was all automated replaces I wouldn't fret too hard...